### PR TITLE
make yellow play button accessible

### PR DESCRIPTION
**Pull request by akash1810**
## What does this change?

the current yellow and white fails accessibility tests, so making the button yellow and grey, with a darkened hover state (per social buttons)

original change https://github.com/guardian/frontend/pull/13471
## What is the value of this and can you measure success?

n/a
## Does this affect other platforms - Amp, Apps, etc?

no
## Screenshots

yellow
## Request for comment

@sammorrisdesign @zeftilldeath

### CODE REVIEW

File: static/src/stylesheets/module/content/_media-player.scss [hunk 1]
```diff
@@ -129,17 +134,12 @@ $ima-controls-height: 70px;
             top: 1.85em;
             border-style: solid;
             border-width: 1em 0 1em 2.4em;
-            border-color: transparent transparent transparent $neutral-7;
+            border-color: transparent transparent transparent #333333;
```
Comment (sndrs) [inline static/src/stylesheets/module/content/_media-player.scss]: this should be a colour variable https://github.com/guardian/frontend/blob/master/static/src/stylesheets/pasteup/palette/_src.scss#L9

Comment (akash1810) [inline static/src/stylesheets/module/content/_media-player.scss]: ah brill, thanks! Changed.

**Comment by sndrs** [comment]
👍

**Comment by ScottPainterGNM** [comment]
@Calanthe I think this has had a knock on effect to the play button we use in the 'Watch Now' so might need another look.

image

**Comment by akash1810** [comment]
@ScottPainterGNM @Calanthe shoot, sorry! Can you send me the URL for this page (again 😊), I can take a look.

**Comment by ScottPainterGNM** [comment]
No worries! It's https://www.theguardian.com/commercial/advertiser-content/renault-car-of-the-future/design-competition-episode1

**Comment by akash1810** [comment]
fixed here - https://github.com/guardian/frontend/pull/13505

**Pull request merged**